cmd/manager/main.go

```diff
@@ -110,17 +111,6 @@ func run() error {
 		return err
 	}
 
-	// Create Service object to expose the metrics port.
-	_, err = metrics.ExposeMetricsPort(ctx, metricsPort)
-	if err != nil {
-		log.V(logtypes.OperatorSDK).Info(err.Error())
-	}
-
-	// start cache and wait for sync
-	log.V(logtypes.OperatorSDK).Info("init chache")
-	cache := mgr.GetCache()
```

Comment:
just for clarification, are we not going to need caching initialization anymore?

Reply:
it seems the caching is working even when removing this command. I think it's enabled by default